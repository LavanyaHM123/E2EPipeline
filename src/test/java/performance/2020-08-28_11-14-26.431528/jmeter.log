2020-08-28 11:15:15,307 INFO o.a.j.u.JMeterUtils: Setting Locale to en_EN
2020-08-28 11:15:15,348 INFO o.a.j.JMeter: Loading user properties from: C:\Users\Prakat-L-0005\.bzt\jmeter-taurus\5.2.1\bin\user.properties
2020-08-28 11:15:15,350 INFO o.a.j.JMeter: Loading system properties from: C:\Users\Prakat-L-0005\.bzt\jmeter-taurus\5.2.1\bin\system.properties
2020-08-28 11:15:15,351 INFO o.a.j.JMeter: Loading additional properties from: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-28_11-14-26.431528\jmeter-bzt.properties
2020-08-28 11:15:15,352 INFO o.a.j.JMeter: Setting System properties from file: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-28_11-14-26.431528\system.properties
2020-08-28 11:15:15,365 INFO o.a.j.JMeter: Copyright (c) 1998-2019 The Apache Software Foundation
2020-08-28 11:15:15,365 INFO o.a.j.JMeter: Version 5.2.1
2020-08-28 11:15:15,365 INFO o.a.j.JMeter: java.version=1.8.0_202
2020-08-28 11:15:15,366 INFO o.a.j.JMeter: java.vm.name=Java HotSpot(TM) 64-Bit Server VM
2020-08-28 11:15:15,366 INFO o.a.j.JMeter: os.name=Windows 10
2020-08-28 11:15:15,366 INFO o.a.j.JMeter: os.arch=amd64
2020-08-28 11:15:15,366 INFO o.a.j.JMeter: os.version=10.0
2020-08-28 11:15:15,366 INFO o.a.j.JMeter: file.encoding=Cp1252
2020-08-28 11:15:15,366 INFO o.a.j.JMeter: java.awt.headless=true
2020-08-28 11:15:15,367 INFO o.a.j.JMeter: Max memory     =1073741824
2020-08-28 11:15:15,367 INFO o.a.j.JMeter: Available Processors =4
2020-08-28 11:15:15,374 INFO o.a.j.JMeter: Default Locale=English (EN)
2020-08-28 11:15:15,374 INFO o.a.j.JMeter: JMeter  Locale=English (EN)
2020-08-28 11:15:15,374 INFO o.a.j.JMeter: JMeterHome=C:\Users\Prakat-L-0005\.bzt\jmeter-taurus\5.2.1
2020-08-28 11:15:15,374 INFO o.a.j.JMeter: user.dir  =C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance
2020-08-28 11:15:15,376 INFO o.a.j.JMeter: PWD       =C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance
2020-08-28 11:15:19,962 INFO o.a.j.JMeter: IP: 192.168.99.1 Name: Prakat-L005 FullName: Prakat-L005
2020-08-28 11:15:19,963 INFO o.a.j.JMeter: user.classpath=C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-28_11-14-26.431528;C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-28_11-14-26.431528
2020-08-28 11:15:19,964 INFO o.a.j.JMeter: Adding to classpath and loader: C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-28_11-14-26.431528
2020-08-28 11:15:19,968 INFO o.a.j.JMeter: Adding to classpath and loader: C:/Users/Prakat-L-0005/Desktop/Automation pipeline/e2e-automation-pipeline-master/src/test/java/performance/2020-08-28_11-14-26.431528
2020-08-28 11:15:19,985 INFO o.a.j.s.FileServer: Default base='C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance'
2020-08-28 11:15:20,031 INFO o.a.j.s.FileServer: Set new base='C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-28_11-14-26.431528'
2020-08-28 11:15:21,096 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2020-08-28 11:15:21,144 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2020-08-28 11:15:21,167 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2020-08-28 11:15:21,211 INFO o.a.j.s.SaveService: Loading file: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-28_11-14-26.431528\modified_requests.jmx
2020-08-28 11:15:21,532 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2020-08-28 11:15:21,540 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2020-08-28 11:15:21,655 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:21,655 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:21,656 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:21,656 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:21,656 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2020-08-28 11:15:21,656 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2020-08-28 11:15:21,940 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2020-08-28 11:15:21,952 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2020-08-28 11:15:21,952 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2020-08-28 11:15:21,971 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2020-08-28 11:15:21,972 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2020-08-28 11:15:24,393 INFO o.a.j.JMeter: Running test (1598593524393)
2020-08-28 11:15:24,619 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : basic-performance-test
2020-08-28 11:15:24,620 INFO o.a.j.e.StandardJMeterEngine: Starting 10 threads for group basic-performance-test.
2020-08-28 11:15:24,620 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2020-08-28 11:15:24,629 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2020-08-28 11:15:24,639 INFO c.b.j.t.c.ConcurrencyThreadStarter: Start supplying threads
2020-08-28 11:15:27,737 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2020-08-28 11:15:27,738 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2020-08-28 11:15:27,739 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2020-08-28 11:15:27,742 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2020-08-28 11:15:27,820 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-1
2020-08-28 11:15:28,888 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = Prakat-L005
2020-08-28 11:15:28,917 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2020-08-28 11:15:31,140 INFO o.a.j.p.h.s.h.LazyLayeredConnectionSocketFactory: Setting up HTTPS TrustAll Socket Factory
2020-08-28 11:15:31,168 INFO o.a.j.u.JsseSSLManager: Using default SSL protocol: TLS
2020-08-28 11:15:31,169 INFO o.a.j.u.JsseSSLManager: SSL session context: per-thread
2020-08-28 11:15:33,884 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-2
2020-08-28 11:15:34,604 INFO o.a.j.u.SSLManager: JmeterKeyStore Location:  type JKS
2020-08-28 11:15:34,608 INFO o.a.j.u.SSLManager: KeyStore created OK
2020-08-28 11:15:34,608 WARN o.a.j.u.SSLManager: Keystore file not found, loading empty keystore
2020-08-28 11:15:36,299 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:36,586 INFO o.a.j.p.h.s.ResourcesDownloader: Creating ResourcesDownloader with keepalive_inseconds : 60
2020-08-28 11:15:38,614 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.CssParser
2020-08-28 11:15:38,614 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.CssParser
2020-08-28 11:15:38,614 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.CssParser
2020-08-28 11:15:40,047 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-3
2020-08-28 11:15:46,167 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-4
2020-08-28 11:15:52,173 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-5
2020-08-28 11:15:58,179 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-6
2020-08-28 11:16:04,186 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-7
2020-08-28 11:16:10,192 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-8
2020-08-28 11:16:16,204 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-9
2020-08-28 11:16:22,209 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test-ThreadStarter 1-10
2020-08-28 11:17:55,314 INFO c.b.j.t.c.ConcurrencyThreadStarter: Done supplying threads
2020-08-28 11:17:55,421 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-5
2020-08-28 11:17:55,421 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-5
2020-08-28 11:17:55,553 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-1
2020-08-28 11:17:55,553 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-1
2020-08-28 11:17:55,652 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-3
2020-08-28 11:17:55,652 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-3
2020-08-28 11:17:56,175 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-6
2020-08-28 11:17:56,176 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-6
2020-08-28 11:17:56,764 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-10
2020-08-28 11:17:56,764 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-10
2020-08-28 11:17:57,249 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-2
2020-08-28 11:17:57,249 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-2
2020-08-28 11:17:57,269 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-8
2020-08-28 11:17:57,269 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-8
2020-08-28 11:17:57,868 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-4
2020-08-28 11:17:57,868 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-4
2020-08-28 11:17:58,299 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-9
2020-08-28 11:17:58,300 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-9
2020-08-28 11:17:58,764 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test-ThreadStarter 1-7
2020-08-28 11:17:58,764 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test-ThreadStarter 1-7
2020-08-28 11:17:58,767 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2020-08-28 11:17:58,782 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
