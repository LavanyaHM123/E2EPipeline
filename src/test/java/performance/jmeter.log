2020-08-28 11:14:55,791 INFO o.j.r.PluginManagerCMD: Command is: install-for-jmx
2020-08-28 11:14:57,598 INFO o.j.r.JARSourceHTTP: Requesting https://jmeter-plugins.org/repo/?installID=docker-90b612754f78d870e7c1f441bc24cf5d-nongui
2020-08-28 11:15:03,446 INFO o.j.r.PluginManager: Plugins Status: [jpgc-functions=2.1, jpgc-casutg=2.9, jpgc-dummy=0.4, jpgc-ffw=2.0, jpgc-fifo=0.2, jpgc-json=2.7, jpgc-prmctl=0.4, jpgc-perfmon=2.1, jpgc-plugins-manager=1.3, jpgc-tst=2.5, jmeter-core=5.2.1, jmeter-ftp=5.2.1, jmeter-http=5.2.1, jmeter-jdbc=5.2.1, jmeter-jms=5.2.1, jmeter-junit=5.2.1, jmeter-java=5.2.1, jmeter-ldap=5.2.1, jmeter-mail=5.2.1, jmeter-mongodb=5.2.1, jmeter-native=5.2.1, jmeter-tcp=5.2.1, jmeter-components=5.2.1]
2020-08-28 11:15:03,452 INFO o.j.r.PluginManagerCMD: Params line is: C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-28_11-14-26.431528\modified_requests.jmx
2020-08-28 11:15:04,261 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2020-08-28 11:15:04,320 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2020-08-28 11:15:04,352 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2020-08-28 11:15:04,551 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2020-08-28 11:15:04,559 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2020-08-28 11:15:05,425 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:05,425 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:05,425 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:05,426 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2020-08-28 11:15:05,426 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2020-08-28 11:15:05,427 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2020-08-28 11:15:05,696 INFO o.j.r.PluginManager: Finishing without changes
