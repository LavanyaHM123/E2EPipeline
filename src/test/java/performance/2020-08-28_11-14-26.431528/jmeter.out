Created the tree successfully using C:\Users\Prakat-L-0005\Desktop\Automation pipeline\e2e-automation-pipeline-master\src\test\java\performance\2020-08-28_11-14-26.431528\modified_requests.jmx
Starting standalone test @ Fri Aug 28 11:15:21 IST 2020 (1598593521865)
Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
Tidying up ...    @ Fri Aug 28 11:17:58 IST 2020 (1598593678779)
... end of run
